13/alpine3.20/Dockerfile

```
@@ -4,6 +4,14 @@
 # PLEASE DO NOT EDIT IT DIRECTLY.
 #
 
+FROM golang:1.23.6 AS gosubuilder
```

If security is the reason for rebuilding, then you probably should always use the latest golang (1.23.10) by removing the patch digits.
In addition, Alpine images should probably use -alpine base images instead of Debian.
```suggestion
FROM golang:1.23-alpine3.21 AS gosubuilder
```

---

Most maintainers in most projects don't allow update contributions as you can see here after your own work.
Most of my pull requests have never been accepted, especially if I added dependabot in addition to all my tedious manual updates (docker, npm, gomod, cargo, github-actions, ...).
I mean there's a reason why there are decades of technical debt in most projects if the maintainers even don't allow dependabot to automate most of that work 🤷

Of course it is nice that dependabot would create those patches / hashes in a timely matter, but maintainers would still not merge them, so what would be gained?

What could work much better for users would be a cron workflow to rebuild the image with all available updates, compare if anything (meaningful) changed and release / publish if necessary without having to wait for a maintainer to press a merge button.
Then consumers would receive the best possible fresh image everyday.
It's hard to conceive why thousands of upstream maintainers work hard everyday to analyze and fix bugs when downstream maintainers just choose to ignore those updates for months or years.